spec/spec_helper.rb

```diff
@@ -0,0 +1,4 @@
+ENV['RAILS_ENV'] ||= 'test'
+
+require File.expand_path '../../config/environment', __FILE__
+require 'rspec/rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.